cbmc/proofs/DeserializeSuback/cbmc-batch.yaml

```diff
@@ -1,4 +1,4 @@
-jobos: ubuntu16
-cbmcflags:        '=--unwindset;_decodeSubackStatus.0:100;--bounds-check;--pointer-check;--unwinding-assertions='
-goto: DeserializeSuback.goto
+cbmcflags: "--flush;--object-bits;8;--unwind;1;--unwindset;valid_IotMqttSubscriptionList.0:2,valid_IotMqttOperationList.0:2,_decodeSubackStatus.0:15;--bounds-check;--pointer-check;--unwinding-assertions;--nondet-static;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-overflow-check;--undefined-shift-check;--signed-overflow-check;--unsigned-overflow-check;--flush"
```

Comment:
Again, I'd remove the `--flush` option.

Reply:
No.  We need to flush output to stdout the moment it is generated so that it appears in the CI log when something is hanging.  We don't want to see something hang and wonder if it has made it into the SAT phase yet.